plugins/poll/assets/javascripts/components/poll-results-number-voters.js.es6

```diff
@@ -3,29 +3,14 @@ import User from 'discourse/models/user';
 import PollVoters from 'discourse/plugins/poll/components/poll-voters';
 
 export default PollVoters.extend({
-  @computed("pollsVoters", "poll.options", "showMore", "isExpanded", "numOfVotersToShow")
-  users(pollsVoters, options, showMore, isExpanded, numOfVotersToShow) {
-    var users = [];
-    var voterIds = [];
-    const shouldLimit = showMore && !isExpanded;
-
-    options.forEach(option => {
-      option.voter_ids.forEach(voterId => {
-        if (shouldLimit) {
-          if (!(users.length > numOfVotersToShow - 1)) {
-            users.push(pollsVoters[voterId]);
-          }
-        } else {
-          users.push(pollsVoters[voterId]);
-        }
-      })
-    });
-
-    return users;
+  @computed("poll.voters", "pollsVoters")
+  canLoadMore(voters, pollsVoters) {
+    return pollsVoters.length < voters;
   },
 
-  @computed("pollsVoters", "numOfVotersToShow")
-  showMore(pollsVoters, numOfVotersToShow) {
-    return !(Object.keys(pollsVoters).length < numOfVotersToShow);
+  @computed("poll.options", "offset")
+  voterIds(options) {
+    const ids = [].concat(...(options.map(option => { return option.voter_ids; })));
```

Comment:
Trivial but you don't need the `{ return x }` in this case:

`const ids = [].concat(...(options.map(option => option.voter_ids)));`